src/features/movies/Genres/styled.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  `;
+
+  export const Genre = styled.p`
+  background-color: ${({ theme }) => theme.color.mystic};
```

Comment:
@Dawid-Zieba you missed "S" in this code, correct it to: `color: ${({theme }) => theme.colors.mystic};` 👍